api/proto/core/pipeline/definition/definition.proto

```diff
@@ -65,7 +65,7 @@ message PipelineDefinition {
   string fileName = 18;
   string pipelineSourceId = 19;
```

Comment:
and this one

Reply:
One PR only does one thing, and the next PR completes the standard naming